packages/app/src/cli/models/app/loader.ts

```diff
@@ -389,6 +389,10 @@ class AppLoader<TConfig extends AppConfiguration, TModuleSpec extends ExtensionS
       usedCustomLayoutForExtensions: configuration.extension_directories !== undefined,
     })
 
+    if (process.env.REMOTE_DOM_EXPERIMENT) {
```

Comment:
use `isTruthy(process.env.REMOTE_DOM_EXPERIMENT)`, otherwise using `REMOTE_DOM_EXPERIMENT=false` would also pass this `if`.

(and following my previous comment about injecting the env -> `isTruthy(this.env.REMOTE_DOM_EXPERIMENT)`)